core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -452,6 +452,28 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         return _getCollateral(rawTotalPositionCollateral);
     }
 
+    function _reduceSponsorPosition(address sponsor, FixedPoint.Unsigned memory ratio) internal {
+        // Rounding issues.
+        if (ratio.rawValue == 1) {
```

Comment:
I agree there could be rounding issues here if the ratio is not exactly 1 after
`tokensToLiquidate.div(positionToLiquidate.tokensOutstanding)`, but perhaps this is not a bug. If this were to happen, the user would unintentionally trigger a "partial liquidation" and have a tiny remaining amount of "sponsor" position?

Reply:
Yeah there'd be like 1 wei or whatever left. Not an actual problem monetarily for anyone, just clutter/dust.